### PR TITLE
added STO title and updated system developer title

### PR DESCRIPTION
**Pull request by ninanorgren**
**Summary of changes**

Fix issue https://app.asana.com/0/1205707359650184/1206055942700380/f 
- Added STO titles to remaining
- Updated to System Developer

### CODE REVIEW

**Comment by ninanorgren** [comment]
Accidentally pushed changes for another issue to this branch also, https://app.asana.com/0/1205707359650184/1206055942392457/f 

Resized the pedagogic support box so the text does not overflow

File: app/views/about/us.html.erb [hunk 1]
```diff
@@ -53,41 +53,41 @@
...
             <div class="col-md-3">
               <div class="image-container">
-                <%= image_tag(asset_path("scilifelab/staff/kristen-resize.png"), alt: 'staff') %>
-                <div class="image-text">
-                  <h4>Kristen Schroeder</h4> <br>
-                  kristen.schroeder@scilifelab.se<br>
-                  Stockholm University <br>
-                  SciLifeLab Solna
-                </div>
+                <div class="image-header">Scientific Training Officers</div>
```
Comment (harshita3199) [inline app/views/about/us.html.erb:71]: remove 's' from Officers to Scientific Training Officer

File: app/views/about/us.html.erb [hunk 2]
```diff
@@ -53,41 +53,41 @@
...
             <!-- Image 3 -->
             <div class="col-md-3">
               <div class="image-container">
-                <%= image_tag(asset_path("scilifelab/staff/jill-resize.png"), alt: 'staff') %>
-                <div class="image-text">
-                  <h4>Jill Jaworski</h4> <br>
-                  jillian.jaworski@scilifelab.se<br>
-                  Stockholm University <br>
-                  SciLifeLab Lund
+                <div class="image-header">Scientific Training Officers</div>
```
Comment (harshita3199) [inline app/views/about/us.html.erb:84]: Scientific Training Officer

**Review by harshita3199 — CHANGES_REQUESTED** [review]
Added the comments to just change from Officers to 'Officer'

**Review by harshita3199 — APPROVED** [review]
Approved

**Pull request merged**